### DISCUSSION TITLE
Cannot see a page with a user or organization name ending in a hyphen in GitHub Mobile for Android

### DISCUSSION THREAD

**Discussion by h3y6e** [community/community#39113] [Mobile] [answered]
### Select Topic Area

Bug

### Body

In GitHub Mobile for Android, when I try to open a page with a user or organization name ending in a hyphen (e.g. https://github.com/camphor-), I get an "Invalid user or organization name" message and cannot see it.

<img src="https://user-images.githubusercontent.com/38322494/201787758-6b2f724a-a061-4711-98f2-4b15dcc0a4e6.png" alt="Screenshot of github mobile for android. Error text is displayed in the center." height="600">

The iOS version displays the page without any problem.

```
GitHub Android v1.86.0 (684); OS SDK v31; Device: Sony XQ-BT44
```

**Comment by stevepopovich** [comment, marked as answer]
Hi @h3y6e! Great catch. I will file an issue with the team and get this looked at. Thanks!

**Reply by wolframhaussig** [reply]
Hi,
Should this be fixed already? I still have the same issue with a hyphen in the name:
https://github.com/Assorted-Development

Best regards
Wolfram

**Reply by stevepopovich** [reply]
This has not been fixed yet, but it is in our backlog.
